client/gui-qt/hudwidget.cpp

```diff
@@ -828,11 +828,10 @@ hud_action::hud_action(QWidget *parent, const QIcon &icon,
  */
 void hud_action::paintEvent(QPaintEvent *event)
 {
-  QRect rx, ry, rz;
+  ;
```

Comment:
??

Reply:
> ??

Unused variable `ry`

Reply:
It seems you introduced a line with just a `;`...